src/screens/General/Leaderboard.js

```diff
@@ -33,7 +34,7 @@ class Leaderboard extends Component {
 		this.props.fetchMembersPoints();
 	}
 
-	_keyExtractor = (item, index) => index;
+	keyExtractor = (item, index) => index;
```

Comment:
This and all other instances of keyExtractor should be put in utils on a file
(not sure what it would be named maybe just general) and just imported wherever used. You don't need to find all places where it's used and change them right now but at least do it with the file's you've changed

Reply:
It's used in so many places, I'll move it after this update

Reply:
You don't need to find it in all the places where it's used but atleast start the trend now with this one